src/rules/typedefWhitespaceRule.ts

```diff
@@ -119,4 +125,15 @@ class TypedefWhitespaceWalker extends Lint.RuleWalker {
         const options = allOptions[0];
         return options[option];
     }
+
+    private positionBeforeColon(node: ts.FunctionDeclaration | ts.FunctionExpression | ts.AccessorDeclaration |
```

Comment:
Let's just simplify this to `node: ts.Node` since we're only using the `getChildren()` method which should be available on all nodes